src/schemas/Clinical/index.ts

```diff
@@ -233,12 +241,9 @@ type ClinicalErrorRecord = {
      message: string;
 }
 
-const convertClinicalDataToGql = (
-  programShortName: string,
-  data: any,
-): ClinicalEntityData => {
-  const clinicalEntities: ClinicalEntityRecord[] = data.clinicalEntities.map((entity: any) => {
-    const records: EntityRecord[][] = entity.records.map((record: any) => (
+const convertClinicalDataToGql = ({programShortName, clinicalEntities}: ClinicalResponseData) => {
+  const clinicalDisplayData: ClinicalEntityRecord[] = clinicalEntities.map((entity: any) => {
```

Comment:
I think `entity: any` can be `entity: EntityDataRecord`

Reply:
I've updated these types, please give a second look